3723/1/page_printer.rb

```diff
@@ -0,0 +1,37 @@
+require_relative 'sorted_pages'
```

Comment:
Why it's required? I see no usage in the class

Reply:
Done